jgclark.DailyJournal/src/journal.js

```diff
@@ -146,7 +134,11 @@ export async function dayReview(): Promise<void> {
       case 'int': {
         const reply = await CommandBar.showInput(questionType[i], `${question[i]}: %@`)
         if (reply != null && isInt(reply)) {
-          reviewLine = questionLines[i].replace(/<int>/, reply)
+          if (questionLines[i].startsWith('-')) {
```

Comment:
I don't think this piece of logic is yet clear in the README.

Reply:
yes - well paid attention :)

Reply:
hmm it is this bulletpoint part:

```
  - you can also add bulletpoints with an idenfitier e.g. `-(thoughts) <string>` -> the identifier doesn't get rendered
  - the purpose of the identifier is to see on which question the user currently is (otherwise one would only have a lot of `-`)
```

i also added the second line because of your first comment